tutorials/plot_epoching_and_averaging.py

```diff
@@ -141,6 +142,16 @@
 evoked_right = epochs['Auditory/Right'].average(picks=picks)
 
 ###############################################################################
+# Notice we have used forward slashes ('/') to separate the factors of the
+# conditions of the experiment. We can use these 'tags' to select for example
+# all left trials (both visual left and auditory right) ...
+
+epochs_left = epochs['Left']
+
+# ... or to select a very specific subset. This is the same as above:
+evoked_left = epochs['Auditory', 'Left'].average(picks=picks)
```

Comment:
surprised this works .. but it is an implicit tuple and we support tuples. interesting.